hw/top_earlgrey/data/ip/chip_pwrmgr_testplan.hjson

```diff
@@ -62,7 +62,10 @@
         "PWRMGR.LOW_POWER.WAKE_INFO"
       ]
       tests: ["chip_sw_pwrmgr_random_sleep_all_wake_ups"]
-      bazel: ["//sw/device/tests:pwrmgr_random_sleep_all_wake_ups"]
+      bazel: [
+        "//sw/device/tests:pwrmgr_random_sleep_all_wake_ups",
+        "//sw/device/tests:pwrmgr_sleep_wake_5_bug_test"
```

Comment:
Should this test have been mapped in ES @engdoreis?

It's a known bug on this branch that will always fail.

Reply:
In that case, we should remove the `silicon_owner_sival_rom_ext` exec_env and unmap it.

Reply:
Okay, I've opened https://github.com/lowRISC/opentitan/pull/23344 to remove the test from this branch entirely. It will persist as a regression test in `master`.

The test isn't useful with the silicon environments removed so I've deleted the whole thing.